job_templates/ci_job.xml.em

```diff
@@ -207,7 +207,19 @@ echo "# BEGIN SECTION: Build Dockerfile"
 @[    if os_name == 'linux-aarch64']@
 docker build --pull ${DOCKER_BUILD_ARGS} --build-arg PLATFORM=aarch64 -t ros2_batch_ci_aarch64 linux_docker_resources
 @[    elif os_name == 'linux-rhel']@
-docker build --pull ${DOCKER_BUILD_ARGS} -t ros2_batch_ci_rhel linux_docker_resources -f linux_docker_resources/Dockerfile-RHEL
+# FIX ME !!
```

Comment:
```suggestion
```

This isn't a hack that we can "fix" and take out. The scare comment doesn't provide any value here. At best, this could be a TODO to revert once EPEL 9 has stabilized.